epub33/rs/index.html

```diff
@@ -1715,6 +1715,17 @@ <h4>The <code>rendition:flow</code> property</h4>
 					<p>Reading systems MUST ignore the <code>rendition:flow</code> property and its overrides when
 						processing <a href="https://www.w3.org/TR/epub-33/#def-layout-pre-paginated">pre-paginated spine
 							items</a> [[epub-33]].</p>
+
+					<p class="note">
+						Reading systems may decide to disregard this restriction, and accept 
```

Comment:
Minor nit, but it's kind of weird to say that RSes will decide things. I'd either drop "decide to" from this sentence or change it to "Reading system developers".

Reply:
Agreed, will change